ftml/src/parsing/rule/impls/block/blocks/anchor.rs

```diff
@@ -23,7 +23,7 @@ use crate::tree::AnchorTarget;
 
 pub const BLOCK_ANCHOR: BlockRule = BlockRule {
     name: "block-anchor",
-    accepts_names: &["a", "anchor"],
+    accepts_names: &["a", "a_", "anchor", "anchor_"],
     accepts_special: true,
```

Comment:
Would it be a good idea / more consistent to, in the future, either:

1. have trailing-underscore tags be some sort of static configuration option e.g. `accepts_modifier`?
2. have leading-asterisk tags be generically declared in `accepts_names`?

If the former the two should probably be named either to make the most sense in their individual functions or to describe what effect they have on the syntax

Reply:
1. Hmm, I suppose that's possible. The way it works for "special" is that `[[*` is a distinct token from `[[`, so it's just a field when block checking happens. So I guess this means the field does the underscore checking itself, before handing off to the block rule. I will think on this.
2. As mentioned above, this would require some changes to how rules are parsed. Additionally it causes issues if there's a space after `[[*` and before the name (though apparently Wikidot itself does not support this behavior).